.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,71 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+name: "CodeQL"
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [master]
+  schedule:
+    - cron: '0 8 * * *'
```

Comment:
Why the cron job? Is this to catch someone bypassing the branch protection rules?

Reply:
It was in be default.

I'm thinking it's more to account for updates to the scanning engine, for projects where you aren't pushing PR's as often.

Happy to reduce the frequency, but I think it should exist for sure. I'm not actually sure how it will run/report to be honest, so call it an experiment.